target/linux/layerscape/patches-5.4/820-usb-0009-usb-dwc3-Add-workaround-for-host-mode-VBUS-glitch-wh.patch

```diff
@@ -44,23 +44,23 @@ Reviewed-by: Peter Chen <peter.chen@nxp.com>
  
 --- a/drivers/usb/dwc3/core.h
 +++ b/drivers/usb/dwc3/core.h
-@@ -1041,6 +1041,8 @@ struct dwc3_scratchpad_array {
+@@ -1045,6 +1045,8 @@ struct dwc3_scratchpad_array {
   * 	2	- No de-emphasis
   * 	3	- Reserved
   * @dis_metastability_quirk: set to disable metastability quirk.
 + * @host_vbus_glitches: set to avoid vbus glitch during
 + *                      xhci reset.
+  * @dis_split_quirk: set to disable split boundary.
```

Comment:
Consider putting dis_split_quirk before host_vbus_glitches here and in the next change, in order to keep local alphabetic order and more reasonable groups.

Reply:
@adschm - Can one do this easily using quilt?  The alternative is to untar the source and edit, then rebuild the patch.  Perhaps there is an easier method?

Reply:
Personally, I would actually just have edited the patch manually, and then run the refresh script on the layerscape target only.

Reply:
Do you mind pasting the manually modified patch as a gist?  I will use it, re-run the script, and rebase/push.

Reply:
I saved your patch (omitting the last line containing the ...) but it did not apply cleanly.

```
% patch -Np1 -i fix.patch
patching file target/linux/layerscape/patches-5.4/820-usb-0009-usb-dwc3-Add-workaround-for-host-mode-VBUS-glitch-wh.patch
Hunk #1 FAILED at 32.
Hunk #2 FAILED at 44.
2 out of 2 hunks FAILED -- saving rejects to file target/linux/layerscape/patches-5.4/820-usb-0009-usb-dwc3-Add-workaround-for-host-mode-VBUS-glitch-wh.patch.rej
```